pages/community/support-enquiry.html

```diff
@@ -0,0 +1,68 @@
+title: Support Enquiry
+description: This document gives you instructions how to provide relevant information for support enquiries when reporting issues with your Crate installation.
+
+{% extends "base.html" %}
+
+{% block body %}
+<div class="wrapper wrapper-bg-primary">
+  <div class="container">
+    <h1>Reporting Issues</h1>
+    <p class="lead">{{ description }}</p>
+  </div>
+</div>
+
+<div class="wrapper wrapper-border">
+  <div class="container">
+      <p>If you have general questions about Crate, e.g. how to use certain Crate specific data types or how to perform fulltext search, etc. just go ahead and drop us a email at <a href="mailto:support@crate.io"><b>support@crate.io</b></a>, join our <a href="https://gitter.com/crate/crate">Gitter chat</a> or chat using the <a href="https://intercom.io">Intercom widget</a> on our website.</p>
+
+    <p>However, if you experience any issues with your already running Crate cluster you can provice certain information that helps us understanding your issues better and therefore improves the solution to your problems.</p>
+    <p>Cause in the end we're happy when our users are happy!</p>
+
+    <p><b>Ideally, your report would include following points (if applicable):</b></p>
+
+    <ul>
+        <li>Crate logs with the relevant lines</li>
+        <li>Stack trace from server or client</li>
+        <li>System/cluster information gathered using <a href="http://pypi.python.org/pypi/crash">Crash</a></li>
+    </ul>
+
+    <p>The first 2 points are pretty much self explanatory. The third one however, can be tricky, especially when you don't know what we're looking for.
```

Comment:
The first two points are self explanatory, but the third point can be difficult, especially when you don't know what you're looking for.